pootle/i18n/formatter.py

```diff
@@ -15,12 +15,14 @@
 from django.utils.translation import get_language, to_locale
 
 
-def get_locale_formats():
-    for language in [get_language(), settings.LANGUAGE_CODE, 'en-us']:
+def get_locale_formats(locale=None):
+    languages = [locale] if locale else []
+    languages += [get_language(), settings.LANGUAGE_CODE, 'en-us', 'en']
+    for language in languages:
         try:
             locale = babel_core.Locale.parse(to_locale(language))
             break
-        except UnknownLocaleError:
+        except (UnknownLocaleError, ValueError):
```

Comment:
Do we want to catch ValueError? This sounds remarkably like `Except` catchall but without info I'm not sure.